### PR TITLE
Emphasized the warning on s3 compatible services for bulk ops

### PR DESCRIPTION
**Pull request by kathancox**
Fixed [DOC-5452](https://cockroachlabs.atlassian.net/browse/DOC-5452)

Added the warning on s3-compatible storage to more places across the bulk ops pages to increase visibility.

### CODE REVIEW

**Comment by kathancox** [comment]
Cloud storage bulk ops page: https://deploy-preview-14937--cockroachdb-docs.netlify.app/docs/stable/use-cloud-storage-for-bulk-operations.html#url-format
Backup: https://deploy-preview-14937--cockroachdb-docs.netlify.app/docs/stable/backup.html

**Comment by kathancox** [comment]
@shermanCRL & @livlobo I've increased the number of warnings across the bulk docs re s3-compatible storage. Let me know what you think. 

One thing I wasn't sure of: I have added this warning to backup, restore, import/into, and export. I felt like that could be a bit over the top, but I did see in Slack there had been issues with import/export with s3 too. I know we mainly want to recommend that users don't put their backup in s3... so we don't have issues with restore. I haven't included changefeeds, since that is one directional.

File: _includes/v22.2/misc/s3-compatible-warning.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
+Unlike Amazon S3, Google Cloud Storage, and Azure storage options, the usage of S3-compatible services is not actively tested by Cockroach Labs.
```
Comment (shermanCRL) [inline _includes/v22.2/misc/s3-compatible-warning.md:2]: Worth calling out examples? The ones we hear bout most are:

MinIO https://min.io
Red Hat Ceph https://docs.ceph.com/en/pacific/radosgw/s3/

**Review by shermanCRL — APPROVED** [review]
Nice improvements, thanks!

File: v21.1/use-cloud-storage-for-bulk-operations.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Azure                                                       | `azure`     | Stor
...
-S3-compatible services                                     | `s3`        | Bucket name                                      | **Warning:** Unlike Amazon S3, Google Cloud Storage, and Azure storage options, the usage of S3-compatible services is not actively tested by Cockroach Labs. <br><br>`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN`, `AWS_REGION`&nbsp;[<sup>3</sup>](#considerations) (optional), `AWS_ENDPOINT`<br><br>For more information, see [Authentication - S3-compatible services](#authentication).   
+S3-compatible services                                     | `s3`        | Bucket name                                      | **Warning:** Unlike Amazon S3, Google Cloud Storage, and Azure Storage options, the usage of S3-compatible services is not actively tested by Cockroach Labs. <br><br>`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN`, `AWS_REGION`&nbsp;[<sup>3</sup>](#considerations) (optional), `AWS_ENDPOINT`<br><br>For more information, see [Authentication - S3-compatible services](#authentication).
```
Comment (kathancox) [inline v21.1/use-cloud-storage-for-bulk-operations.md:34]: I accidentally pulled this in with a find and replace. My intention wasn't to update v21.1 docs, therefore no further updates included for v21.1 in this PR

File: _includes/v21.2/misc/s3-compatible-warning.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
+Unlike Amazon S3, Google Cloud Storage, and Azure Storage options, Cockroach Labs does **not** actively test [S3-compatible services](use-cloud-storage-for-bulk-operations.html) (e.g., [MinIO](https://min.io/), [Red Hat Ceph](https://docs.ceph.com/en/pacific/radosgw/s3/)).
```
Comment (andf-crl) [inline _includes/v21.2/misc/s3-compatible-warning.md:2]: Nit: "Unlike A, B, and C, D is a different letter." But here D is not "Cockroach Labs", but "S3-compatible services". Switching from `unlike` to `while` helps here:

> While Cockroach Labs actively tests Amazon S3, Google Cloud Storage, and Azure Storage options, we **do not** test third-party [S3-compatible services](use-cloud-storage-for-bulk-operations.html) such as [MinIO](https://min.io/) and [Red Hat Ceph](https://docs.ceph.com/en/pacific/radosgw/s3/).

File: v22.1/backup.md [hunk 4]
```diff
@@ -24,22 +24,24 @@ You can also backup:
...
 - Exclude a table's row data from a backup using the [`exclude_data_from_backup`](take-full-and-incremental-backups.html#exclude-a-tables-data-from-backups) parameter.
 - `BACKUP` is a blocking statement. To run a backup job asynchronously, use the `DETACHED` option. See the [options](#options) below.
 
-{{site.data.alerts.callout_success}}
-To view the contents of an Enterprise backup created with the `BACKUP` statement, use [`SHOW BACKUP`](show-backup.html).
-{{site.data.alerts.end}}
+### Storage considerations
+
+- [HTTP storage](use-a-local-file-server-for-bulk-operations.html) is not supported for `BACKUP` and `RESTORE`.
+- Modifying backup files in the storage location could invalidate a backup, and therefore, prevent a restore. In v22.1 and later, **we recommend enabling [object locking](use-cloud-storage-for-bulk-operations.html#object-locking) in your cloud storage bucket.**
```
Comment (andf-crl) [inline v22.1/backup.md:43]: Just commenting that lines 27-28 and 40-43 don't appear to be part of this effort. If you meant to do these, carry on!

**Review by andf-crl — APPROVED** [review]
LGTM, but suggest a slight rework of the alert message. Also see updates to two files that appear unrelated, left a comment, but please disregard if you meant to make them as part of other update efforts. Thanks!

File: v22.1/backup.md (hunk 4 above)
Comment (kathancox) [reply v22.1/backup.md:43]: that was just a move around to create a separate space for the storage considerations, thanks for checking.

**Pull request merged**